localplanning.services/src/components/applications/ApplicationViewer.module.css

```diff
@@ -0,0 +1,28 @@
+/* Overrides to style injected HTML */
+.viewer-overrides {
+  :global h1 {
```

Comment:
`:global()` is required here as we're trying to style pre-compiled HTML here, without additional scoped classes a CSS module would normally add.